tests/googlePayments.test.js

```diff
@@ -0,0 +1,93 @@
+// @TODO Move the below to a global setup file
+const chai = require('chai');
+const sinon = require('sinon');
+const iapModule = require('in-app-purchase');
+const sinonStubPromise = require('sinon-stub-promise');
+sinonStubPromise(sinon);
```

Comment:
this should be separated from the require statements, same for expect = chai.expect